code/arith.py

```diff
@@ -6,28 +7,35 @@ def main():
 
     operator = sys.argv[1]
     assert operator in ['add', 'subtract', 'multiply', 'divide'], (
-        'Operator is not one of add, subtract, multiply, or divide: '
-        'bailing out')
+        '''Operator is not one of add, subtract, multiply, or divide:
+        bailing out''')
     try:
         operand1, operand2 = float(sys.argv[2]), float(sys.argv[3])
     except ValueError:
         print('Cannot convert input to a number: bailing out')
-        return
+        return None
 
     do_arithmetic(operand1, operator, operand2)
+    return None
```

Comment:
Since we explain in the Functions episode that "functions are not required to include a return statement", and I don't think it is common practice to always put one explicitly at the end of a function, my preference would be not to add these to all of the functions.

Reply:
I've already provided my thoughts regarding this in the comment above. I'm not sure how to proceed further.